### ISSUE TITLE
Add new card with drawboard

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#1633]
Originally reported on Google Code with ID 730

```
Allow users to create an image to a new card by just drawing/writing with the touchscreen.
It enables fast card addition on-the-spot.
Also, it might be more efficient than alternatives to add a small diagram.
```

Reported by `nicolas.raoul` on 2011-08-16 07:03:33

**Comment by Olifair** [comment]
For me an OCR is not as important as to be able to create simply cards that can include a simple drawing (basically using the same code that is already included for the testing). Should OCR not be a different feature request ?

**Comment by eginhard** [comment]
@Olifair There already is a separate issue for OCR (#1899). It was just mentioned above to allow easier searching for this issue.

**Comment by mikehardy** [comment]
In the absence of a PR or any interest currently I'm going to close this. Seems like an image-oriented app that could slice-and-dice images, OCR them, or allow drawing and used the API to add cards would be well-received...

**Comment by jeffchang5** [comment]
I'm interested in building this :)

**Comment by mikehardy** [comment]
@jeffchang5 cool! Check out the apisample app and how it works in combo with the AnkiDroid app via an API on the phone - if you can do a small (to start) app that does some image processing and then uses the API to add, it might work...https://github.com/ankidroid/apisample

**Comment by jeffchang5** [comment]
Is there no intention of using handwriting as a first party feature?

**Comment by mikehardy** [comment]
you should ask on the mailing list - there are several ways to do handwriting input as far as I can tell, that behave pretty well from what I understand?

**Comment by jeffchang5** [comment]
@mikehardy, imho, if this feature exists, there could be better ways to surface this feature. As a younger developer, I can't discern where this feature exists from the documentation and traversing the settings. I recently got a Note 10 and wanted to this feature for language learning which is what I imagine a lot of people downloading this app is using it for.

**Comment by robertocarta** [comment]
> ```
> you should ask on the mailing list - there are several ways to do handwriting input as far as I can tell, that behave pretty well from what I understand?
> ```

What ways? I couldn't find any

**Comment by david-allison-1** [comment]
@robertocarta 

This feature is implemented in our [alpha builds for Anki 2.16](https://docs.ankidroid.org/#betaTesting)

Otherwise, many keyboards for Android support a handwriting recognition mode.

**Comment by robertocarta** [comment]
@david-allison-1 handwriting recognition is very different from drawing with a stylus. When studying maths, for instance, quickly drawing math symbols makes a lot more sense.

**Comment by david-allison-1** [comment]
This feature is implemented in our [alpha builds for Anki 2.16](https://docs.ankidroid.org/#betaTesting)

#9363 #9394
